cmd/runner/config.go

```diff
@@ -9,23 +10,38 @@ import (
 )
 
 type AgentConfig struct {
-	API    APIConfig    `yaml:"api"`
-	Runner RunnerConfig `yaml:"runner"`
+	API     APIConfig     `yaml:"api"`
+	Runner  RunnerConfig  `yaml:"runner"`
+	Logging LoggingConfig `yaml:"logging,omitempty"`
 }
 
-func NewAgentConfig(t runner.Token) *AgentConfig {
-	return &AgentConfig{
+func NewAgentConfig(t runner.Token, platform string) (c *AgentConfig, err error) {
```

Comment:
in particular, I wonder if we could rejig the config format so that api token and resource class are next to each other, and then have this script only spit out those 2 lines - leaving the docs (or a future installer) as the source-of-truth for the rest